mobile/src/libs/enums/navigation/root-screen-name.enum.ts

```diff
@@ -3,6 +3,8 @@ const RootScreenName = {
   SIGN_IN: 'Sign In',
   SIGN_UP: 'Sign Up',
   SURVEY: 'Survey',
+  SETTINGS: 'Settings',
+  MY_PROFILE: 'My Profile',
```

Comment:
Profile should be a separate stack screen with home screen and notifications screen
```suggestion
  PROFILE: 'Profile',
```

Reply:
@Liza-Veis could you please explain, should i create separate navigation stack and add there profile and notifications (notifications like on image ?) ? because we have Home like bottom tab and whole main component includes all bottom tabs. 
![Снимок экрана 2023-09-07 141205](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/111873795/c85bcb8c-09cf-4135-b8a0-c545902a52ac)

Reply:
no, notifications is the screen where you will get after pressing first item (notifications and reminders ) on user profile, we don't need to add one more button to header